tests/sentry/api/endpoints/test_project_user_issue.py

```diff
@@ -0,0 +1,169 @@
+from __future__ import annotations
```

Comment:
and can we also move this test to the associated tests/sentry/issues/endpoints folder? 🙏🏼